DER/ExchangeTradedDerivatives/Futures.rdf

```diff
@@ -592,8 +424,28 @@
 		<skos:definition xml:lang="en">How many of the underlying securities can be sold.</skos:definition>
 	</owl:DatatypeProperty>
 	
-	<owl:Class rdf:about="https://spec.edmcouncil.org/fibo/ontology/MD/DerivativesTemporal/FuturesTemporal/FuturesTradingAccountHolder">
-		<rdfs:subClassOf rdf:resource="&fibo-fnd-agr-ctr;Counterparty"/>
+	<owl:Class rdf:about="&fibo-fbc-fi-fi;Future">
```

Comment:
The fact that this ontology is adding all these restrictions is a red flag - consider whether they should either be in the original ontology or this ontology should have a subclass.

Reply:
Removed the restrictions, which were somewhat redundant and in some cases overly restrictive, and moved the subclass relationship between Future and ForwardContract to the Forwards ontology for further discussion.